lib/matchers/duration.js

```diff
@@ -6,8 +6,17 @@ var TYPE_ERROR = 'should be a duration string (e.g. \"10s\")';
 module.exports = function (opts) {
 
   if (!opts) opts = {};
-  var minValue = (typeof opts.min === 'string') ? ms(opts.min) : 0;
-  var maxValue = (typeof opts.max === 'string') ? ms(opts.max) : Number.MAX_VALUE;
+
+  var minValue = (opts.min != null) ? ms(opts.min) : 0;
+  var maxValue = (opts.max != null) ? ms(opts.max) : Number.MAX_VALUE;
+
+  if (typeof minValue !== 'number') {
```

Comment:
I think you're right, since we've already checked that `opts.min` itself is a string.
It only leaves 2 options: `ms('5s') == 5000`, and `ms('5foo') == undefined`.

I guess this is different from the validation at [line 26](https://github.com/TabDigital/strummer/pull/16/files#diff-35b56a11dd5425feea24130f5ef2cd4aR26), since we haven't checked if `value` is a string (and `ms(50) == '50s'`, and `ms(null) == 'nullms'`.... :confounded:).